### PR TITLE
Remove leftover IDelete uses (Py)

### PR DESCRIPTION
**Pull request by joshmoore**
A few references were still left to the deprecated then removed IDelete service. Here we:

 * fix reference to queueDelete in `tables.py`
 * remove 'broken' marker from `test_services.py` cc: @ximenesuk 
 * also remove `DeleteService` from `gateway/__init__.py` cc: @will-moore 
 * comments out a failing [assert](https://github.com/openmicroscopy/openmicroscopy/pull/3729/files#diff-fa6bb86e1444246dbf41f9ac2fcd9e32R155) cc: @mtbc 

This primarily needs to keep tests green and have the API changes reviewed. [This test](https://ci.openmicroscopy.org/view/Failing/job/OMERO-5.1-merge-integration-broken/lastCompletedBuild/testReport/test.integration.tablestest.test_service/TestTables/testDelete/) should drop off the "broken radar" though.

### CODE REVIEW

**Comment by joshmoore** [comment]
This also fixes https://trac.openmicroscopy.org.uk/ome/ticket/12577

File: components/tools/OmeroPy/src/omero/tables.py [hunk 1]
```diff
@@ -846,24 +847,30 @@ def update(self, data, current=None):
     def delete(self, current=None):
         self.assert_write()
         self.close()
-        prx = self.factory.getDeleteService()
-        dc = omero.api.delete.DeleteCommand(
+        dc = omero.cmd.Delete(
```
Comment (ximenesuk) [inline components/tools/OmeroPy/src/omero/tables.py]: `Delete2` ?

**Comment by ximenesuk** [comment]
Other than the minor comments the changes look good and the tests are passing

**Comment by joshmoore** [comment]
Pushed, @ximenesuk

**Comment by ximenesuk** [comment]
+1 Just need to confirm tests on next integration build.

**Comment by joshmoore** [comment]
No table tests failure recently:
https://ci.openmicroscopy.org/job/OMERO-5.1-merge-integration-python/428/testReport/test.integration.tablestest.test_service/TestTables/
Closing.

**Pull request merged**